rest/serializers.py

```diff
@@ -97,3 +102,65 @@ class Meta:
             'id',
             'slug',
         )
+
+
+class LearningResourceSerializer(ModelSerializer):
+    """Serializer for LearningResource"""
+
+    learning_resource_type = SerializerMethodField()
+
+    @staticmethod
+    def get_learning_resource_type(obj):
+        """Instead of returning an id return the name for this enum"""
+        return obj.learning_resource_type.name
+
+    class Meta:
+        # pylint: disable=missing-docstring
+        model = LearningResource
+        fields = (
+            'id',
+            'learning_resource_type',
+            'static_assets',
+            'title',
+            'description',
+            'content_xml',
+            'materialized_path',
+            'url_path',
+            'parent',
+            'copyright',
+            'xa_nr_views',
+            'xa_nr_attempts',
+            'xa_avg_grade',
+            'xa_histogram_grade',
+        )
+        read_only_fields = (
```

Comment:
Can `read_only_fields` just be set to `= fields`, since they're the same? It would be easier to keep them in sync.